roles/lib_openshift/library/oc_route.py

```diff
@@ -1704,18 +1690,6 @@ def run_ansible(params, check_mode=False):
                          'content': params['key_content'],
                          'value': None, }, }
 
-        if params['tls_termination'] and params['tls_termination'].lower() != 'passthrough':  # E501
-
-            for key, option in files.items():
-                if key == 'destcacert' and params['tls_termination'] != 'reencrypt':
-                    continue
-
-                option['value'] = OCRoute.get_cert_data(option['path'], option['content'])  # E501
```

Comment:
```python
  if params['tls_termination'] and params['tls_termination'].lower() != 'passthrough':  # E501

            for key, option in files.items():
                if key == 'destcacert' and params['tls_termination'] != 'reencrypt':
                    continue

                option['value'] = OCRoute.get_cert_data(option['path'], option['content'])  # E501

                if not option['value']:
                    return {'failed': True,
                            'msg': 'Verify that you pass a value for %s' % key}
```
If the `tls_termination` is not `passthrough` then we need the certificate data.
if the `key` is destcacert and `tls_termination` is not `reencrypt`, then we skip.  This only skips `destcacert` if `tls_termination` is not reencrypt`, right?  

What about `cacert`, `key`, and `cert`?